Run-SystemTest.ps1

```diff
@@ -25,12 +25,12 @@ Process {
         Push-Location $contractFirstSysTestDir
         
         Exec { & dotnet add package Workleap.OpenApi.MSBuild --prerelease --source $outputDir }
-        Exec { & dotnet build -c Release }
+        Exec { & dotnet build -c Release -warnaserror }
```

Comment:
is this something that could be relevant for our other building blocks?

Reply:
I don't think so since it a problem specific that it is a MsBuild task and no other building block are. Also we have created a story so this library take into account the flag correctly so eventually we will remove this.